core/runtime/src/main/java/io/quarkus/runtime/configuration/QuarkusConfigFactory.java

```diff
@@ -26,14 +26,7 @@ public SmallRyeConfig getConfigFor(final SmallRyeConfigProviderResolver configPr
             //TODO: this code path is only hit when start fails in dev mode very early in the process
             //the recovery code will fail without this as it cannot read any properties such as
             //the HTTP port or logging info
-            return configProviderResolver.getBuilder().forClassLoader(classLoader)
-                    .addDefaultSources()
-                    .addDefaultInterceptors()
-                    .addDiscoveredSources()
-                    .addDiscoveredConverters()
-                    .addDiscoveredInterceptors()
-                    .withProfile(ProfileManager.getActiveProfile())
-                    .build();
+            return ConfigUtils.emptyConfigBuilder().addDefaultSources().addDiscoveredSources().build();
```

Comment:
Is it intentional to no longer add default/discovered converters and interceptors here?

Reply:
The call to `ConfigUtils.emptyConfigBuilder()` provides a builder with discovered converters enabled, so need to do it again.